### ISSUE TITLE
Duplicator not working on server

### ISSUE THREAD

**Issue by logan2611** [wiremod/advdupe2#39]
Ok so I am running a spacebuild server and I keep getting this message everytime I try to paste something that is saved:
```
[ERROR] addons/advdupe2/lua/advdupe2/cl_networking.lua:282: Calling net.Start with unpooled message name [http://goo.gl/qcx0y]
  1. Start - [C]:-1
   2. SendFileToServer - addons/advdupe2/lua/advdupe2/cl_networking.lua:282
    3. Function - addons/advdupe2/lua/advdupe2/cl_networking.lua:290
     4. unknown - lua/includes/modules/usermessage.lua:87
```
And I get this everytime I try to copy and paste something:
```
[AdvDupe2Notify]	addons/advdupe2/lua/advdupe2/sv_clipboard.lua:1029: attempt to perform arithmetic on field 'SpawnRate' (a nil value)
Warning: Unhandled usermessage 'Undone'
```
And after I try I try to paste something it says "Advanced Duplicator 2 is busy".
Adv Dupe 2 works on all my other servers, but I have yet to restart my other servers in ages.
The original adv dupe works fine on this server.

**Comment by Divran** [comment]
This error indicates that there is another error during startup. Change map on your server and look in the server's console for errors during startup.

Specifically in this file: https://github.com/wiremod/advdupe2/blob/58cf63b971aee7cd0e8881680525156c9ba5bb6a/lua/autorun/server/advdupe2_sv_init.lua#L149 because this is where the "unpooled message name" becomes pooled.

**Comment by logan2611** [comment]
I will do that

**Comment by logan2611** [comment]
Not finding any errors related to Adv Dupe 2 :/

**Comment by Jack37** [comment]
I've had the same problem recently. Photon: Lighting Engine [Beta] addon was causing it, so it's most likely another addon messing with adv dupe 2 there.

**Comment by AbigailBuccaneer** [comment]
This is hopefully fixed by #40 - can you test again @logan2611?

**Comment by logan2611** [comment]
I fixed it by recreating my server.
But if it happens again I will try this out.
